playwright/src/fixtures/content.page.ts

```diff
@@ -131,15 +133,19 @@ class ContentPage {
   async clickHighlight(highlight_id: string) {
     // Click on a highlight
     // param: highlight_id of the highlight to be clicked
-    this.highlightIdlocator = this.page.locator(`[data-highlight-id="${highlight_id}"][data-highlighted="true"]`)
-    await this.highlightIdlocator.click()
+    const highlightIdlocator = await this.page.$$(`[data-highlight-id="${highlight_id}"][data-highlighted="true"]`)
+
```

Comment:
@gregfitch The `highlightIdlocator` defined in line 136 returns undefined. what am I doing wrong here.